samples/posix/gettimeofday/README.rst

```diff
@@ -0,0 +1,32 @@
+.. _posix-gettimeofday-sample:
+
+POSIX gettimeofday() with clock initialization over SNTP
+########################################################
+
+Overview
+********
+
+This samples demonstrates POSIX gettimeofday() function working "out of box"
```

Comment:
How about:
```
This  sample application demonstrates using the POSIX
gettimeofday() function to display the absolute wall clock
time every second. At system startup, the current time is
queried using the SNTP networking protocol, enabled by
setting the :option:`CONFIG_POSIX_CLOCK_SNTP_INIT`
and :option:`CONFIG_POSIX_CLOCK_SNTP_INIT_SERVER`
options.
```

Reply:
> How about:

Updated to use the suggested paragraph. @dbkinder, please re-review.